src/Mimi.jl

```diff
@@ -721,6 +722,38 @@ end
 
 #End of graph section
 
-include("references.jl")
+# Begin plotting section
+
+"""
+Three defaults: single line plot, multiple line plots with legend, bar graph
+when indexing over regions, etc.
+"""
+function auto_plot(m::Model, component::Symbol, parameter::Symbol ; index::Symbol = :time, legend::Symbol = nothing, xlabel = string(index), ylabel = string(parameter), dest = nothing)
+  if isnull(m.mi)
+      run(m)
+  end
```

Comment:
Alright, that sounds good. What is the best way to check if a model has been run? I believe that m.mi will be instantiated in the case where m has been built but not run, though I may be mistaken.